apps/src/JavaScriptModeErrorHandler.js

```diff
@@ -0,0 +1,132 @@
+/** @file Decorator and delegator for error, warning and info events in toolkits
+  * that use droplet and allow students to write and execute JavaScript. */
+import annotationList from './acemode/annotationList';
+import logToCloud from './logToCloud';
+
+/** @enum {string} */
+export const LogLevel = {
+  ERROR: 'LogLevel/ERROR',
+  WARNING: 'LogLevel/WARNING'
+};
```

Comment:
`makeEnum` has been bugging me because RubyMine doesn't recognize or autocomplete the enum values, which it does when you define it manually.  It's probably still the right approach though; I'll update this.  I wonder if there's some JSDoc magic that would help here.